VaticanBAV/et/BAVet131.xml

```diff
@@ -27,52 +27,1023 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <title type="complete" xml:lang="gez" ref="LIT3308Nicene"/>
+                                    <textLang mainLang="gez"/>
+                                </msItem> 
+                                <msItem xml:id="ms_i2.2">
+                                    <locus from="75v" to="76r"/>
+                                    <title type="complete" xml:lang="gez">ጸሎተ፡ ሃይማኖት፡ ዘ፲ወ፪ ሐዋርያት፡</title>
+                                    <note>The text is also found in <ref type="work" corresp="LIT1104Anapho"/>, cp. <bibl><ptr target="bm:Harden1928Anaphoras"/>
+                                        <citedRange unit="page">52-54</citedRange></bibl>.</note>
+                                    <textLang mainLang="gez"/>
+                                </msItem> 
+                                <msItem xml:id="ms_i2.3">
+                                    <locus from="76r" to="76v"/>
+                                    <title type="complete" xml:lang="gez">Prayer of the Sunday</title>
+                                    <textLang mainLang="gez"/>
+                                </msItem> 
+                                <msItem xml:id="ms_i2.4">
+                                    <locus from="76v" to="77r"/>
+                                    <title type="complete" xml:lang="gez" ref="LIT4942Sarawita"/>
+                                    <textLang mainLang="gez"/>
+                                    <incipit xml:lang="gez">ሠራዊተ፡ መላእክቲሁ፡ ለመድኃኔ፡ ዓለም፡</incipit>
+                                    <note>
+                                        After being repeated in four musical tones (indicated with the four abbreviations <foreign xml:lang="gez">ማቱ፡</foreign>, <foreign xml:lang="gez">ነቁ፡</foreign>, 
+                                        <foreign xml:lang="gez">ቁራጸሩ፡</foreign>, <foreign xml:lang="gez">ኮቦ፡</foreign>), the chant contains antiphons for the following 
+                                        feast days: for the Nativity, for the sabbaths, for St Mary, for the cherubs, for the Apostles, for the prophers, the righteous, the martyrs, the angels.
+                                    </note>
+                                </msItem> 
+                                <msItem xml:id="ms_i2.5">
+                                    <locus from="77r" to="77v"/>
+                                    <title type="complete" xml:lang="gez">Prayers after the Communion</title>
+                                    <textLang mainLang="gez"/>
+                                    <note>As in <ref type="work" corresp="LIT1104Anapho"/>, cp. <bibl><ptr target="bm:Mercer1915Liturgy"/>
+                                        <citedRange unit="page">364-365, 372 ll. 18-29</citedRange></bibl>.</note>
+                                </msItem>                                 
+                            </msItem> 
+                            
+                            <msItem xml:id="ms_i3">
```

Comment:
At least ms_i3–ms_i12 are part of what normally appears in the Meeraf and could perhaps be encoded as such